podpac/core/data/types.py

```diff
@@ -560,8 +560,88 @@ def get_band_numbers(self, key, value):
 
         return matches
 
+class SpecifyCoordinatedMixin(tl.HasTraits):
```

Comment:
I'll take suggestions on a better name here.

Reply:
I wouldn't worry too much, it is fairly internal. You have typo, should be Coordinates not Coordinated?

But I would choose something more like `DatasetCoordinatesMixin` or `LoadCoordinatesMixin`, since you aren't actually specify the coordinates.